source/mutex/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mutex"
-version = "0.1.0"
+version = "1.0.0"
```

Comment:
so, my question is...do we really need to ship a 1.0 for this crate? It's the `mutex-traits` crate that contains the core abstractions that need to be stable, and I don't think version churn for this crate is as big an issue. On the other hand, maybe we should just go to 1.0 and churn major versions --- that's the way semver is _meant_ to be used --- but folks might interpret a major version as a stability commitment that this crate doesn't _need_ to make...

Reply:
I'll add a note in the docs for this, we should definitely be clear about it.

Reply:
Lemme know if you think https://github.com/tosc-rs/scoped-mutex/pull/14/commits/244c02fa5f9594a0ffc48ca8f8015c6bff2e4e37 covers it!

Reply:
Oh also, yeah, my preference is for many `N.0.0` releases for `mutex`, instead of many `0.N.0` release.